src/stories/spinner.stories.js

```diff
@@ -0,0 +1,24 @@
+import React from "react"
+import Generator from "./generator.js"
```

Comment:
we should be able to drop the `.js` extension from the import. really not a biggie - just for the sake of consistency with the other imports
```suggestion
import Generator from "./generator"
```